### PR TITLE
Fix : run-test.sh for studyViewerConfig

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

**Review by themarcelor — APPROVED** [review]
+1

File: run-tests.sh [hunk 1]
```diff
@@ -385,9 +385,9 @@ if [ -z "$checkForPresenceOfMetadataIngestionSowerJob" ]; then
 fi
 
 # studyViewer
-if [[ $(curl -s "$portalConfigURL" | jq 'contains({studyViewerConfig}) | not') == "true" ]] || [[ ! -z "$testedEnv" ]]; then
+if [[ $(curl -s "$portalConfigURL" | jq 'contains({studyViewerConfig}) | not') == "true" ]]; then
```
Comment (haraprasadj) [inline run-tests.sh:388]: the double quotes around true should be removed, in other similar checks too

**Review by haraprasadj — APPROVED** [review]

File: run-tests.sh (hunk 1 above)
Comment (haraprasadj) [reply run-tests.sh]: that did not seem to work, so trying removing `== true` part

**Review by haraprasadj — APPROVED** [review]

**Review by themarcelor — APPROVED** [review]
+1

**Pull request merged**